makefiles/shared.mk

```diff
@@ -81,4 +81,8 @@ oas-page-build:
 	node ${OAS_MODULE_PATH} --bundle ${BUNDLE_PATH} --output ${REPO_DIR}/public --redoc ${REDOC_PATH} --repo ${REPO_DIR} --site-url ${URL}/${MUT_PREFIX}
 
 persist-data:
-	node ${PERSISTENCE_MODULE_PATH} --path ${BUNDLE_PATH}
\ No newline at end of file
+	ifeq ($(USE_PERSISTENCE), true)
+		-node ${PERSISTENCE_MODULE_PATH} --path ${BUNDLE_PATH}
```

Comment:
Should this be `-node` or `node`?

Reply:
this was for [ignoring errors ](https://www.gnu.org/software/make/manual/html_node/Errors.html#:~:text=To%20ignore%20errors%20in%20a,to%20the%20shell%20for%20execution.&text=This%20causes%20make%20to%20continue,unable%20to%20remove%20a%20file) but will double check for execution

Reply:
@rayangler  https://workerpoolstaging-qgeyp.mongodbstitch.com/pages/job.html?collName=dotcom_queue&jobId=63bdba5eb9c1638e2c81c235

this is from a bad commit, but the build was still pushed as the last step